outputs.tf

```diff
@@ -0,0 +1,8 @@
+output "public_ip" {
+  value = aws_instance.public.public_ip
+
+}
+
+output "private_ip" {
+  value = aws_instance.private.private_ip
+}
```

Comment:
Add newline at the end of file.

Reply:
I have added the new line. 
Can I know why we add new line i.e what errors may occur ?

Reply:
okay